plugins/inputs/aerospike/aerospike.go

```diff
@@ -68,6 +102,14 @@ func (a *Aerospike) Gather(acc telegraf.Accumulator) error {
 		a.initialized = true
 	}
 
+	if a.NumberHistogramBuckets == 0 {
+		a.NumberHistogramBuckets = 10
+	} else if a.NumberHistogramBuckets > 100 {
+		a.NumberHistogramBuckets = 100
+	} else if a.NumberHistogramBuckets < 1 {
+		a.NumberHistogramBuckets = 10
+	}
```

Comment:
You inherited this, but there's an opportunity to clean it up. plugins can have an Init() function that allows you to do init and config related things that runs once and before the first Gather call.

Just add `func (a *Aerospike) Init() error {` and you can move all this setup-related stuff there.

Reply:
maybe next time we do more work, we'll update this. Did not have time this iteration